src/styles/baseThemes/darkBaseTheme.js

```diff
@@ -19,5 +19,6 @@ export default {
     disabledColor: ColorManipulator.fade(Colors.fullWhite, 0.3),
     pickerHeaderColor: ColorManipulator.fade(Colors.fullWhite, 0.12),
     clockCircleColor: ColorManipulator.fade(Colors.fullWhite, 0.12),
+    shadowColor: Colors.fullWhite,
```

Comment:
Well, have a look at the dark theme section of https://www.google.com/design/spec/style/color.html#color-themes.